src/components/main/note/Note.tsx

```diff
@@ -45,6 +46,7 @@ const SmallActionButton = styled(IconButton)(() => ({
   borderRadius: '50%',
   position: 'absolute',
   scale: [0.5, 0.5, 1],
+  fontSize: '0rem', // Ensures button remains square.
```

Comment:
Interesting ! I used `span`s to wrap elements and keep them square. If this works too then 🆗 !

Reply:
In my case, `span` did not work, thus I fixed the font size to 0. Works perfectly 💯